BREAKING.md

```diff
@@ -23,25 +23,29 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 - [Remove `type` field from `ShareLinkInfoType`](#Remove-type-field-from-ShareLinkInfoType)
 - [Remove `ShareLinkTypes` interface](#Remove-ShareLinkTypes-interface)
 - [Remove `enableShareLinkWithCreate` from `HostStoragePolicy`](#Remove-enableShareLinkWithCreate-from-HostStoragePolicy)
+- [Add assertion that closes container in DeltaManager if sending an op while concurrently processing another op](#add-assertion-that-closes-container-in-DeltaManager-if-sending-an-op-while-concurrently-processing-another-op)
```

Comment:
I don't believe this needs a note in BREAKING.md. As a consumer of the framework, I'm not entirely sure what to do with this information which sounds like something internal to the framework.

Reply:
I felt that it deserved to stay here as a note that this change will not be optional in the future and the appropriate tests will need to be fixed.

Reply:
Can you rephrase this in a way that's relevant to the customer? I'm reading this and seeing some thing about the DeltaManager which I'm not actually using directly and I'm not sure how it would affect my code or my designs.